woocommerce/woocommerce-gateway-stripe
# Add StripeLink to WooCommerce Checkout block

Fixes #2381

## Changes proposed in this Pull Request:

Introduce StripeLink into the WooCommerce Checkout block.

## Testing instructions

- Navigate to `Settings` -> `Payment Methods` and enable `Link` payment method.
- Install the https://github.com/woocommerce/woocommerce-blocks plugin.
- Create a new page and Checkout block.
- Verify that StripeLink is working correctly in the block.
---


**Post merge**
